students/JasneetChandok/Lesson04/trigrams.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+
+"""Lesson 4: Trigrams Lab
+
+Note: Have refrenced another classmates solution
+
+"""
+
+import random
+
+punctuations = "!" + "-" + "," + "." + "(" + ")" + '"' + '?' + ';' + ":" + '\n'
+replacements = " " * 11
+
+
+def read_file(filename):
+    words = []
+    with open(filename, "r") as text:
+        for line in text.readlines():
+            raw_line = line
+            clean_string = raw_line.translate(raw_line.maketrans(punctuations,replacements))
```

Comment:
Nice clean up here!

Instead of creating your own punctuations string above, you can use a predefined string.punctuation:


import string
string.punctuation #this is a string that contains all that you would need